source/firmware/xilinx-tools/flashing.md

```diff
@@ -2,11 +2,15 @@
 
 During development, JTAG will be the primary interface for programming and debugging the AMDC firmware. Once code is stable, an image can be programed into the AMDC non-volatile memory (NVM). This allows board to boot itself when powered up.
 
-The following steps outline how to create a boot image and flash the AMDC NVM.
+The following steps outline how to create a boot image and flash the AMDC NVM. 
+
+This process varies slightly between single core, and dual core projects. Differences will be noted in the appropriate steps.
+
+Going through this process for the first time, the "create boot image" dialogs may differ slightly from the ones shown below. Use good judgement here and it should be evident what the appropriate steps are. Once you have created an initial boot image - even if it is incorrect and needs to be fixed, you can then match the dialogs exactly to the ones shown below. 
```

Comment:
I was confused by this at first, but then I recalled how the SDK works... When I read it again, it made total sense.

So, not sure if this needs to change to be more clear, or not... The SDK is just really confusing with setting up this dialog!